src/java.base/share/classes/sun/security/pkcs/PKCS7.java

```diff
@@ -266,6 +266,14 @@ private void parseNetscapeCertChain(DerValue val)
         }
     }
 
+
+    //    SignedData ::= SEQUENCE {
+    //        version CMSVersion,
+    //        digestAlgorithms DigestAlgorithmIdentifiers,
+    //        encapContentInfo EncapsulatedContentInfo,
+    //        certificates [0] IMPLICIT CertificateSet OPTIONAL,
+    //        crls [1] IMPLICIT RevocationInfoChoices OPTIONAL,
+    //        signerInfos SignerInfos }
```

Comment:
We should include RFC 5652 here as the file is named PKCS7 which is in RFC 2315 which has different field names and types.